### PR TITLE
Application dashboard

### PR DESCRIPTION
**Pull request by katiejiang**
## Description
Added application dashboard at `/application`.
Renamed `ApplicantProfile` to `Application`.

## Checklist
- [ ] I ran `python manage.py format` to autoformat my code according to the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by katiejiang** [comment]
@stevenbursztyn this is the table you'll need for your task this week!

File: app/models/user.py [hunk 1]
```diff
@@ -204,8 +204,8 @@ def generate_fake(count=100, **kwargs):
                 confirmed=True,
                 role=role,
                 **kwargs)
-            if u.is_applicant():
-                u.applicant_profile = ApplicantProfile.generate_fake()
+            if u.role.id == 1:
```
Comment (stevenbursztyn) [inline app/models/user.py]: can we case on the enum val itself? instead of ==1?

Comment (katiejiang) [reply app/models/user.py]: 👍 good catch

File: app/application/views.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+from app import db
+from app.models import Application, User
+
+application = Blueprint('application', __name__)
+
+@login_required
+@application.route('/')
+def index():
+    print(current_user.role_id)
```
Comment (stephanieyshi) [inline app/application/views.py]: remove this? 😛

Comment (katiejiang) [reply app/application/views.py]: nah

**Comment by katiejiang** [comment]
@stephanieyshi decorators are after route I believe. Haven't tested the decorators yet - we can do that in a separate PR. I want to merge this to unblock Steven 😛

**Comment by stephanieyshi** [comment]
can we merge this?

**Pull request merged**